metagraph/src/graph/representation/bitmap/dbg_bitmap.cpp

```diff
@@ -16,14 +15,14 @@ DBGBitmap::DBGBitmap(size_t k, bool canonical_mode)
       : k_(k),
         canonical_mode_(canonical_mode),
         seq_encoder_(),
-        kmers_(std::pow(static_cast<long double>(seq_encoder_.alphabet.size()), k_) + 1, true),
+        kmers_((1llu << (k * seq_encoder_.bits_per_char)) + 1, true),
```

Comment:
we should have an assert here to ensure that `seq_encoder_.alphabet.size() == 2`

Reply:
Why? Alphabet size can be arbitrary.

Reply:
Never mind, for some reason I was thinking that `KmerExtractor2Bit` was only implemented for DNA alphabets

Reply:
Yeah, the name is confusing. We should change it later